Pegasus/sources/Particle.cpp

```diff
@@ -5,15 +5,19 @@
 * implies agreement with all terms and conditions of the accompanying
 * software licence.
 */
-#include "pegasus/Particle.hpp"
+#include <pegasus/Particle.hpp>
 
 #include <limits>
 #include <stdexcept>
 
 pegasus::Particle::Particle()
-    : m_damping(1.0)
+    : m_position(0, 0, 0)
```

Comment:
I think or you can pass just 0?

Reply:
Yes, I can, not much difference though.

Reply:
same "taste question"

Reply:
I think actually that we should prefer this way, rather than one you proposed. Since it gives a more clear hint on what type it is, without even reading the name of the variable.